demos/machine_translation_demo/python/machine_translation_demo.py

```diff
@@ -230,6 +242,9 @@ def sentences():
             stop = time.perf_counter()
             print(translation)
             logger.info(f"time: {stop - start} s.")
+            if args.output:
+                with open(args.output, 'a', encoding='utf8') as f:
+                    f.write(translation + '\n')
```

Comment:
```suggestion
                    print(translation, file=f)
```

(it's more consistent with the other `print`)